runtime-config-linux.md

```diff
@@ -170,6 +204,27 @@ For example, to run a new process in an existing container without updating limi
 
 #### CPU
 
+`cpu` represents the cgroup subsystems `cpu` and `cpusets`.
+For more information, see [the cpusets cgroup man page](https://www.kernel.org/doc/Documentation/cgroups/cpusets.txt).
+
+The following parameters can be specified to setup the cpu controller:
+
+* **`shares`** *(uint64, optional)* - specifies a relative share of CPU time available to the tasks in a cgroup
+
+* **`quota`** *(uint64, optional)* - specifies the total amount of time in microseconds for which all tasks in a cgroup can run during one period (as defined by **`period`** below)
+
+* **`period`** *(uint64, optional)* - specifies a period of time in microseconds for how regularly a cgroup's access to CPU resources should be reallocated (cfs scheduler only)
```

Comment:
The [CFS docs][1] use caps (“CFS”, not “cfs”).  And I think we should link those scheduler docs here, so maybe:

> … ([CFS scheduler][CFS] only).
>
> [CFS]: https://www.kernel.org/doc/Documentation/scheduler/sched-design-CFS.txt

[1]: https://www.kernel.org/doc/Documentation/scheduler/sched-design-CFS.txt